lms/djangoapps/instructor/views/data_access.py

```diff
@@ -99,6 +100,14 @@ def save_group_name(group_id, group_name):
     return True
 
 
+@transaction.commit_manually
+def flush_transactions():
```

Comment:
As discussed, if django gets upgraded to 1.6 and above, may need to doublecheck autocommit transaction behavior.